algolia/algoliasearch-sfcc-b2c
# feat: Implementing content search frontend

## Content Search UI

![Image2024-01-29 at 10 40 42](https://github.com/algolia/algoliasearch-sfcc-b2c/assets/36648230/71507f17-baa1-4731-a99b-cc3d2b59fb06)

![Image2024-01-29 at 10 41 09](https://github.com/algolia/algoliasearch-sfcc-b2c/assets/36648230/ca983991-53b6-4de4-b24f-b8544ccbc3e8)


Created  UI  Search by utilizing autocomplete and instant search

### Changes

- Add content search functionality for search and autocomplete
- Added preference for managing content search
- Added SSR for Content Search


### Tests

- Run the content indexing ob 
- Enable Content Search
- Check if content results are listed on autocomplete
- Check if content results are listed for search queries
- Check if  content results are listed for category page (should not)
- Check if content results are listed for search queries while there is no content result (should not)
